docs/web/index.md

```diff
@@ -585,10 +587,11 @@ or even another instance of the service.
 Most likely, this will happen when a business entity is stored based on a request and loaded later
 for further processing by a scheduler or due to a new user interaction.
 In this case, the `DetachedMetadataContext` must be persisted along with the entity and recreated
-when the entity is loaded.
-The `DetachedMetadataContext` can be defined as field in any MongoDB entity.
-
-Services that handle requests or messages in parallel must transfer the metadata context to their
-`Runnable` or `Callable` with `MetadataContext.transferMetadataContext(…)`.
-In most cases, developers should prefer`ContainerRequestContextHolder.transferRequestContext(…)`,
-which also transfers the metadata context.
\ No newline at end of file
+when the entity is loaded. The `DetachedMetadataContext` can be defined as field in any MongoDB
+entity.
+So, for services that handle requests or messages in parallel, the
```

Comment:
I would keep beginning a new paragraph here as it is a different topic than persistence.